File: pset6/mario/more/mario.py
#!/usr/bin/env python3
# -*- encoding: utf-8 -*-

# BUG: For some unknown reason there's one more space then required

from cs50 import get_int

height = 0

while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    print(" " * (height - i - 1), end="")
    print("#" * (i + 1))

File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int h = 0; // Stair height
    string stairs = "########";
    string space = "        ";

    // Check for valid input in range
    while (h < 1 || h > 8)
    {
        h = get_int("Height: ");
    }

    // Print stairs
    int c = h - 1; // Counter
    for (int i = 1; i < h + 1; i++)
    {
        printf("%.*s", c, space);
        printf("%.*s", i, stairs);
        printf("\n");
        c--;
    }
}